src/api/admin.ts

```diff
@@ -129,6 +129,20 @@ export interface VenueInput_v2
   end_utc_seconds?: number;
 }
 
+export interface WorldProps {
+  id: string;
+  name: string;
+  description?: string;
+  subtitle?: string;
+  bannerImageFile?: FileList;
+  bannerImageUrl?: string;
+  logoImageFile?: FileList;
+  logoImageUrl?: string;
+  rooms?: Room[];
```

Comment:
should be no rooms as discussed?

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2295/commits/563960b617e371ddfc9d465b3428bdae64fafb0d